jsontableschema/constraints/__init__.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
```

Comment:
Adding note here that applies to al the checks: you either raise, or return the implicit `None`. Returning `None` for a successful check is counter-intuitive IMHO. I think you should return True.